pkg/kubectl/cmd/util/factory_builder.go

```diff
@@ -108,12 +103,6 @@ func (f *ring2Factory) PrintObject(cmd *cobra.Command, isLocal bool, mapper meta
 	_, typer := f.objectMappingFactory.Object()
 	gvks, _, err := typer.ObjectKinds(obj)
 
-	// fall back to an unstructured object if we get something unregistered
-	if runtime.IsNotRegisteredError(err) {
```

Comment:
I remember this being important for handling cases where you querying aggregated types and CRDs.

Reply:
I now handle unknown types in the fundamental scheme safely for ObjectKinds.